packages/default-theme/components/SwProductCarousel.vue

```diff
@@ -33,65 +26,21 @@ export default {
   },
   data() {
     return {
-      products: [
-        {
-          title: 'Cream Beach Bag',
-          image: '/img/productA.png',
-          price: { regular: '50.00 $' },
-          rating: { max: 5, score: 4 },
-          isOnWishlist: true
+      products: {},
+      searchCriteria: {
+        sort: {
+          field: 'price',
+          desc: false
         },
-        {
-          title: 'Cream Beach Bag',
-          image: '/img/productB.png',
-          price: { regular: '50.00 $' },
-          rating: { max: 5, score: 4 },
-          isOnWishlist: false
-        },
-        {
-          title: 'Cream Beach Bag',
-          image: '/img/productC.png',
-          price: { regular: '50.00 $' },
-          rating: { max: 5, score: 4 },
-          isOnWishlist: false
-        },
-        {
-          title: 'Cream Beach Bag',
-          image: '/img/productA.png',
-          price: { regular: '50.00 $' },
-          rating: { max: 5, score: 4 },
-          isOnWishlist: false
-        },
-        {
-          title: 'Cream Beach Bag',
-          image: '/img/productB.png',
-          price: { regular: '50.00 $' },
-          rating: { max: 5, score: 4 },
-          isOnWishlist: false
-        },
-        {
-          title: 'Cream Beach Bag',
-          image: '/img/productC.png',
-          price: { regular: '50.00 $' },
-          rating: { max: 5, score: 4 },
-          isOnWishlist: false
-        },
-        {
-          title: 'Cream Beach Bag',
-          image: '/img/productA.png',
-          price: { regular: '50.00 $' },
-          rating: { max: 5, score: 4 },
-          isOnWishlist: false
-        },
-        {
-          title: 'Cream Beach Bag',
-          image: '/img/productB.png',
-          price: { regular: '50.00 $' },
-          rating: { max: 5, score: 4 },
-          isOnWishlist: false
+        pagination: {
+          page: 1,
+          limit: 10
         }
-      ]
+      }
     }
+  },
+  async mounted() {
+    this.products = await getProducts(this.searchCriteria)
```

Comment:
why do you need to have searchCriteria inside `data`?  They are not going to change :)
Also please consult with @mkucmus searchQuery details to avoid this:
![image](https://user-images.githubusercontent.com/13100280/73348999-afc40280-428a-11ea-811c-c16faeba7e2b.png)